packages/create-next-app/templates/app-tw/js/app/page.js

```diff
@@ -0,0 +1,117 @@
+import Image from 'next/image'
+import { Inter } from 'next/font/google'
+
+const inter = Inter({ subsets: ['latin'] })
+
+export default function Home() {
+  return (
+    <main className="flex min-h-screen flex-col items-center justify-between p-24">
+      <div
+        className={`z-10 w-full max-w-5xl items-center justify-between font-mono text-sm lg:flex`}
```

Comment:
Some inconsistency in " vs {`

Reply:
Changed it everywhere to `"`, except where we use a variable class name for fonts.